packages/toolkit/assistive/form-field-notification.css

```
@@ -0,0 +1,273 @@
+  --_notification-text-sm: 0.75rem;
+  --_notification-text-base: 1rem;
+  --_notification-line-height-sm: 1rem;
+  --_notification-line-height-base: 1.5rem;
+  --_notification-weight-medium: 500;
+
+  --_notification-border-width-default: 1px;
+  --_notification-border-style-default: solid;
+  --_notification-border-radius-default: 0.5rem;
+  --_notification-box-shadow-default: none;
+
+  --_notification-token-current-color: currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)